tasks/preload.js

```diff
@@ -295,7 +302,7 @@ module.exports = function (grunt) {
 
 					var content = JSON.stringify(preloadObject, null, '\t');
 					if (typeof preloadInfo.processContent === 'function') {
-						content = preloadInfo.processContent(content);
+						content = preloadInfo.processContent(content, preloadObject);
```

Comment:
We should now better only pass the `preloadObject` in here and both implementations should do the JSON.stringify on their own.